crates/re_viewer_context/src/viewer_context.rs

```diff
@@ -35,6 +37,10 @@ pub struct ViewerContext<'a> {
     /// Mapping from class and system to entities for the store
     pub entities_per_system_per_class: &'a EntitiesPerSystemPerClass,
```

Comment:
Nit: this needs a more specific name.

Reply:
which is why it is renamed in the next pr 😄